packages/app-project/src/components/AuthModal/components/LoginForm/components/Form/Form.spec.js

```diff
@@ -1,25 +1,31 @@
 import React from 'react'
-import { shallow } from 'enzyme'
-import { expect } from 'chai'
+import { mount, shallow } from 'enzyme'
 import sinon from 'sinon'
-import { TextInput } from 'grommet'
+import { Grommet, TextInput } from 'grommet'
+import zooTheme from '@zooniverse/grommet-theme'
 import {
   userNameFieldId,
   passwordFieldId,
   Form
 } from './Form'
 
-describe('Component > Form', function () {
+describe('LoginForm > Component > Form', function () {
   it('should render without crashing', function () {
-    const wrapper = shallow(<Form />)
+    const wrapper = shallow(
+      <Form />,
+      { wrappingComponent: <Grommet />, wrappingComponentProps: { theme: zooTheme } }
```

Comment:
How weird that they chose to do this instead of
```js
{ wrappingComponent: Grommet, wrappingComponentProps: { theme: zooTheme } }
```
or
```js
{ wrappingComponent: <Grommet theme={zooTheme} /> }
```

Reply:
I don't have an explanation for this either. When using shallow, I could only get it to work if using the JSX component, `<Grommet />` in the options object, but using mount it only works with `Grommet`. That seems like a bug to me.